modules/introduction/partials/new_features-76_4.adoc

```diff
@@ -30,6 +30,16 @@ curl --get -u <username:password> \
 [#new-features-764-search-service]
 === Search Service
 
+* The Search Service now supports pre-filtering on vector search queries. 
+Using pre-filtering as part of your vector search offers two key advantages:
++
+--
+** Enhanced precision and relevance in the search results.
+** Performance optimization.
```

Comment:
Ok so it's not a performance optimization - because we'd be running the search request in 2 phases for this.

What pre-filtering gives is the ability to be able to run vector search over a subset of the indexed data. The "filter" is responsible for qualifying this subset of data.

Reply:
Does that mean we should remove `performance optimization` from the main pre-filtering page?

https://docs.couchbase.com/server/current/vector-search/pre-filtering-vector-search.html#about-pre-filtering

Reply:
Okay, so what advantage does this give the user?

Reply:
No, I just put the benefit in my comment above^.

Reply:
No worries; I'll figure it out.